qa/src/test/resources/features/datastore/DatastoreNewIndexCustomPrefix.feature

```diff
@@ -0,0 +1,49 @@
+###############################################################################
+# Copyright (c) 2018 Eurotech and/or its affiliates and others
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     Eurotech - initial API and implementation
+#
+###############################################################################
+@datastore
```

Comment:
Is the word `datastore` non-capital on purpose?

Reply:
All feature file tags for scenarios are in lower case.